jest.setup.js

```diff
@@ -0,0 +1 @@
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
As I know for the latest Jest better to shorten it to `import '@testing-library/jest-dom';`